AirCasting/CreateSessionViews/ConnectingABView.swift

```diff
@@ -11,7 +11,8 @@ import CoreBluetooth
 struct ConnectingABView: View {
     
     var bluetoothManager: BluetoothManager
-    var selecedPeripheral: CBPeripheral
+    var selectedPeripheral: CBPeripheral
+    let baseURL: BaseURLProvider
```

Comment:
can those be `private`?

Reply:
'ConnectingABView' initializer is inaccessible due to 'private' protection level' this is the comment which I get whenever I try to do it